src/components/Intercom/Intercom.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useCallback } from 'react'
+import IntercomWidget from 'decentraland-dapps/dist/components/Intercom'
+import { getAnalytics, getAnonymousId } from 'decentraland-dapps/dist/modules/analytics/utils'
+import { config } from 'config'
+
+const APP_ID = config.get('INTERCOM_APP_ID', '')
+
+export const Intercom: React.FC = () => {
+  const [dclAnonymousUserID, setDclAnonymousUserID] = useState('')
+
+  const analytics = getAnalytics()
+
+  const analyticsReadyCallback = useCallback(() => {
+    const dclAnonymousUserID = getAnonymousId()
+    if (dclAnonymousUserID) {
+      setDclAnonymousUserID(dclAnonymousUserID)
+    }
+  }, [])
+
+  analytics.ready(analyticsReadyCallback)
```

Comment:
Does it matters if the ready value is set multiple times?
```suggestion
  useEffect(() => analytics.ready(analyticsReadyCallback), [])
```

Reply:
@LautaroPetaccio -- thanks for the review! It does not affect `segment+intercom` functionality but should only be set once when the page is first loaded. It'll update the code